libs/libtirpc/Makefile

```diff
@@ -0,0 +1,66 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=libtirpc
+PKG_VERSION:=1.0.2
+PKG_RELEASE:=1
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL:=git://git.linux-nfs.org/projects/steved/libtirpc.git
+PKG_SOURCE_VERSION:=503ac2e9fa569d95e366766202a7ca840e28b28a
+PKG_SOURCE_SUBDIR:=$(PKG_NAME)-$(PKG_VERSION)-$(PKG_RELEASE)
+PKG_SOURCE:=$(PKG_SOURCE_SUBDIR).tar.gz
+PKG_BUILD_DIR:=$(BUILD_DIR)/$(PKG_SOURCE_SUBDIR)
+PKG_FIXUP:=autoreconf
+PKG_LICENSE:=BSD-4c
+PKG_BUILD_PARALLEL:=1
+PKG_INSTALL:=1
+
+PKG_CONFIG_DEPENDS := \
+	CONFIG_LIBTIRPC_GSSAPI \
+	CONFIG_LIBTIRPC_KRB5
+
+include $(INCLUDE_DIR)/package.mk
+
+LIBTIRPC_DEPENDS:=
+CONFIGURE_ARGS += --disable-static
+ifeq ($(CONFIG_LIBTIRPC_GSSAPI),y)
+	ifeq ($(CONFIG_LIBTIRPC_KRB5),y)
+		LIBTIRPC_DEPENDS += +krb5-libs +e2fsprogs
```

Comment:
I'm unsure whether `e2fsprogs` should be a `DEPENDS` or if it should be `PKG_BUILD_DEPENDS` for `libcom_err`